src/LanguageServer.php

```diff
@@ -190,6 +191,7 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->projectIndex = new ProjectIndex($sourceIndex, $dependenciesIndex, $this->composerJson);
             $stubsIndex = StubsIndex::read();
             $this->globalIndex = new GlobalIndex($stubsIndex, $this->projectIndex);
+            $initializationOptions = $initializationOptions ?? new Options;
```

Comment:
[`initializationOptions`](https://github.com/JSteitz/vscode-php-intellisense/blob/74be99d15703b16b810334b76e53491bc1bc4a4c/src/extension.ts#L101) can be completely different to [`didChangeConfiguration`](https://github.com/JSteitz/vscode-php-intellisense/blob/74be99d15703b16b810334b76e53491bc1bc4a4c/src/extension.ts#L97).
And actually they are not the same in this PR. While `initializationOptions` returns only defined options in the extension, `didChangeConfiguration` will return the whole section with the section name as main key.


**initializationOptions**
```
{
  "fileTypes": [
    ".php"
  ]
}
```

**didChangeConfiguration**
```
{
  "php": {
    "executablePath": "\/usr\/local\/opt\/php71\/bin\/php",
    "fileTypes": [
      ".php"
    ],
    "suggest": {
      "basic": true
    },
    "validate": {
      "enable": true,
      "executablePath": "\/usr\/local\/opt\/php71\/bin\/php",
      "run": "onSave"
    }
  }
}
```